src/pages/case-studies-and-core-tools.tsx

```diff
@@ -1,24 +1,40 @@
+/* eslint-disable react-hooks/rules-of-hooks */
+
+import { getEntryKeys, groups } from "@/data/case-studies";
+import { useEffect, useState, useMemo } from "react";
+import { useAppContext } from "@/contex";
 import Group from "@/components/Group";
-import groups from "@/data/case-studies";
-import HightlightCaseStudyDetails from "@/components/HightlightCaseStudyDetails";
-import { levels } from "@/data/case-studies/levels";
-import { technologies } from "@/data/case-studies/technologies";
+import HighlightCaseStudyDetails from "@/components/HightlightCaseStudyDetails";
+
 const caseStudiesAndCoreTools = () => {
+  const [highlightedElement, setHighlightedElement] = useState<string>("");
+  const { setEntryKeys } = useAppContext();
```

Comment:
The logic used in highlighting an element in the co-design journey should be the same as that used for case studies highlight

Reply:
changed